### ISSUE TITLE
Translate JSOB column that use store_accessor

### ISSUE THREAD

**Issue by darkcode85** [shioyama/mobility#172]
- Rails: 5.2.0.rc1
- Mobility: 0.4.2

In a rails app I have a model called `Setting` which looks like this:

```
create_table :settings do |t|
  t.string :type
  t.jsonb :values, null: false, default: {}
  t.timestamps
end

add_index  :site_settings, :values, using: :gin
```

The values column save multiple values using `store_accessor` so the model look like this:

```
class Setting < ApplicationRecord
  store_accessor :values, :setting_1, :setting_2, :setting_3, :setting_4
end
````

Until that point all is working as expected, but now I need each of the values to be translatable so I tried to add `Mobility` to the model:

```
class Setting < ApplicationRecord
  extend Mobility
  translates :values

  store_accessor :values, :setting_1, :setting_2, :setting_3, :setting_4
end

``` 

But now each time that I want to access to any store_accessor `(Setting.new.setting_1)` this error happen: `NoMethodError (undefined method `[]' for nil:NilClass)`

There is any way to translate `store_accessor` values in the same column? if that is not possible which will be the best approach to translate `store_accessor` values using Mobility?

-- UPDATED

Ah! this is my Mobility configuration:

```
Mobility.configure do |config|
  config.default_backend = :jsonb
  config.accessor_method = :translates
  config.query_method = :i18n
end
```

**Comment by darkcode85** [comment]
I'm trying the same approach with the new backend option `container`, this is my `Gemfile`:

`gem 'mobility', git: 'https://github.com/shioyama/mobility', branch: 'master'`

Then when I try `Setting.new. setting_1 ` I get this error: `NameError (uninitialized constant Mobility::Backends::ActiveRecord::Container)`

@shioyama any help?

**Comment by shioyama** [comment]
For container, you need to be using Mobility version >= 0.4.

But you won't be able to use `store_accessor` with translated values, since AR will access the original column. You *can* translate hashes, but you'll need to define your own accessors.

Any other questions, please post to SO as the issue template suggests.

**Comment by shioyama** [comment]
Just FYI, there was indeed an issue with the container backend in 0.4.2, see #173. It's fixed now in 0.4.3 so if you're interested in using it, please update! :smile:
